fundamentals/fundamentals-bio/src/main/scala/izumi/functional/bio/BIORoot.scala

```diff
@@ -69,6 +70,7 @@ sealed trait BIORootInstancesLowPriority5 extends BIORootInstancesLowPriority6 {
 }
 
 sealed trait BIORootInstancesLowPriority6 {
+  @inline implicit final def BIOMonix: Predefined.Of[BIOAsync[bio.IO]] = Predefined(BIOAsyncMonix)
```

Comment:
I think you should move the BIOConvert3To2 instance ta lower priority trait `BIORootInstancesLowPriority7` now, so that these two instances do not interfere